main/main.go

```diff
@@ -0,0 +1,30 @@
+package main
```

Comment:
The binary is named after the directory containing the main package. Why not leave this at top-level for now?

Reply:
Ah, didn't know you could have a package "main" in a directory not called "main". Fixed.